conf/messages.cy

```diff
@@ -335,7 +335,7 @@ uploadFile.originalC88 = the import declaration, also known as the C88 or SAD
 uploadFile.originalE2 = the entry acceptance, also known as the E2
 uploadFile.originalCalculations = a document or scanned image showing how you calculated the tax or duty that should have been paid
 uploadFile.importAndEntry = the amendment import declaration (C88) and entry acceptance (E2)
-uploadFile.airwayBill = the airway bill
+uploadFile.airwayBill = the air waybill
```

Comment:
changes to welsh file will need removing when welsh translation is merged

Reply:
updated